src/rollup/types.d.ts

```diff
@@ -314,8 +314,13 @@ export interface OutputBundle {
 	[fileName: string]: OutputAsset | OutputChunk;
 }
 
+export interface FilePlaceholder {
+	isPlaceholder: true;
```

Comment:
seems like this is not used anywhere.
can remove in favor of `type: placeholder`